Open Judge System/Web/OJS.Web/Areas/Administration/Views/Problems/EditorTemplates/ProblemGroupsCascadeDropDown.cshtml

```diff
@@ -0,0 +1,31 @@
+﻿@model int?
+
+@using Resource = Resources.Areas.Administration.Problems.Views.Partials.ProblemsPartials
+
+<script>
+    function setContestIdToGetProblemGroups() {
+        return {
+            contestId: $('#ContestId').val()
+        }
+    }
+</script>
+
+@(Html
+      .Kendo()
+      .DropDownList()
+      .Name("ProblemGroupId")
+      .OptionLabel(Resource.Copy_in_new_problem_group)
+      .HtmlAttributes(new { @class = "test-file-dropdown" })
+      .DataTextField("Name")
+      .DataValueField("Id")
+      .DataSource(data =>
+      {
+          data
+              .Read(read => read.Action("GetCascadeProblemGroupsFromContest", "KendoRemoteData")
```

Comment:
Why don't you move the `.Action` method call on a new line as it is a fluent API call.